content/rs/administering/designing-production/performance/shard-placement.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: Configuring Shard Placement
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+{{< embed-md "shard-placement-intro.md"  >}}
+
+## Default Shard Placement Policy
+
+The cluster configuration includes a default policy for the shard placement policy.
+When you create a database, this default policy is applied to the new database.
+The initial policy for all new clusters is `dense`.
+
+You can see the current default shard placement policy in `rladmin info cluster`:
```

Comment:
s/in/by running/

Reply:
To see the current default shard placement policy, run `rladmin info cluster`: